packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -208,6 +202,9 @@ export const UpdateProjectConnection: FC<{
                 name: data.name,
                 dbt: data.dbtConnection,
                 warehouse: data.warehouseConnection,
+                dbtVersion: data?.dbtVersion
+                    ? (data?.dbtVersion as SupportedDbtVersions)
+                    : undefined,
```

Comment:
isn't this the same as `dbtVersion: data?.dbtVersion` ?

Reply:
data?.dbtVersion could be "", from the select (we can't have `undefined` on selects)